app/manifest/v3/_base.json

```diff
@@ -50,7 +50,9 @@
     "http://localhost:8545/",
     "file://*/*",
     "http://*/*",
-    "https://*/*"
+    "https://*/*",
+    "ws://*/*",
+    "wss://*/*"
```

Comment:
does this need to be added to manifest v2 as well?

Reply:
Good call out, This is now fixed. Thanks